Equilobe.DailyReport.Models/ReportPolicy/SourceControlOptions.cs

```diff
@@ -11,8 +11,7 @@ public class SourceControlOptions
     {
         public SourceControlType Type { get; set; }
         public string RepoOwner { get; set; }
-        public string RepoUrl { get; set; }
-        public string RepoName { get; set; }
+        public string Repo { get; set; }
```

Comment:
repo zice exact ce e, e repository, ori ca e url, ori ca e name, conteaza de type
s-a schimbat pentru a facilita binding-ul si verificarile in angular, avand in vedere ca se foloseste aceeasi clasa pentru ambele tipuri